src/com/operators_loops/CountCustomersDeposit.java

```diff
@@ -1,2 +1,29 @@
-package com.operators_loops;public class CountCustomersDeposit {
+package com.operators_loops;
+
+import java.util.Scanner;
+import java.math.*;
```

Comment:
тут краще не використовувати вайлд іморт. Тобто кожен імпорт окремо.
```suggestion
import java.math. BigDecimal;
```

Reply:
четвертий рядочок наче зайвий